packages/design/src/physics/geometry/IBoxGeometry.ts

```diff
@@ -0,0 +1,12 @@
+import { Vector3 } from "@galacean/engine-math";
+import { IGeometry } from "./IGeometry";
+
+/**
+ * Interface for box geometry.
+ */
+export interface IBoxGeometry extends IGeometry {
```

Comment:
IBoxColliderShape should be reused or PhysXBoxGeometry should be used directly, which should not increase the access cost of the physical backend.

Reply:
done